HLTriggerOffline/Btag/interface/HLTBTagHarvestingAnalyzer.h

```diff
@@ -48,6 +49,8 @@ class HLTBTagHarvestingAnalyzer : public DQMEDHarvester {
 			// Histogram handler
 			std::map<std::string, MonitorElement *> H1_;
 
+			std::vector<std::string> modules_;
```

Comment:
Yes, but in this file the variable modules_ is used in the dqmEndJob function only, so it is called only once per job and I guess this change won't improve performance